console/packages/starwhale-core/src/widget/hooks/useRestoreState.tsx

```diff
@@ -56,15 +56,10 @@ export default function useRestoreState<T>(store: StoreType, initialState: T, dy
             if (isOrigin) data = tranformState(data)
 
             Object.keys(data?.widgets).forEach((id) => {
-                _.set(data.widgets, id, replacer(PANEL_DYNAMIC_MATCHES).toOrigin(data.widgets[id], dynamicVars))
+                const origin = replacer(PANEL_DYNAMIC_MATCHES).toOrigin(data.widgets[id], dynamicVars)
+                _.set(data.widgets, id, origin)
             })
 
-            if (store.getState().time >= data?.time) {
```

Comment:
I am not sure if it will bring some side effects, and I will keep paying attention to it.